src/lib.rs

```diff
@@ -281,8 +281,12 @@ pub struct NtpResult {
     pub seconds_fraction: u32,
     /// Request roundtrip time in microseconds
     pub roundtrip: u64,
-    /// Offset of the current system time with one received from a NTP server in microseconds
+    /// Estimated difference between the NTP reference and the system time in microseconds
     pub offset: i64,
+    /// Clock stratum of NTP server
+    pub ref_stratum: u8,
+    /// Nominal precision of NTP server (seconds)
+    pub ref_precision: f32,
```

Comment:
Since the precision response is a 8-bit signed integer, let's leave it as is without float conversion as it leads to build error for `no_std` crate version.